### PR TITLE
[Testing] Added integration test for wires part of #1475

### PR DESCRIPTION
**Pull request by markoer**
Added integration tests for #1475.

@nicolatimeus, please review.

### CODE REVIEW

**Comment by MMaiero** [comment]
@markoer Do you plan another PR in order to integrate those tests in the build? I think, maybe I'm wrong, that we still need a change to the tests pom in order to execute the latest added tests.

**Comment by markoer** [comment]
@MMaiero You're right. I added it now.

File: kura/test/org.eclipse.kura.wire.h2db.component.provider.test/src/main/java/org/eclipse/kura/internal/wire/h2db/store/test/H2DbWireRecordStoreTest.java [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+        int count = resultSet.getInt(1);
+        assertEquals("Unexpected number of records", maxSize, count);
+
+        // store a few records
+        for (int i = 0; i < 5; i++) {
+            dbstore.onWireReceive(wireEvelope);
+
+            // prevent key violations - timestamp only has a 1 ms resolution
+            try {
+                Thread.sleep(1);
```
Comment (nicolatimeus) [inline kura/test/org.eclipse.kura.wire.h2db.component.provider.test/src/main/java/org/eclipse/kura/internal/wire/h2db/store/test/H2DbWireRecordStoreTest.java]: Same as above

File: kura/test/org.eclipse.kura.wire.h2db.component.provider.test/src/main/java/org/eclipse/kura/internal/wire/h2db/store/test/H2DbWireRecordStoreTest.java [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+        wireRecords.add(record);
+        WireEnvelope wireEvelope = new WireEnvelope(emitterPid, wireRecords);
+
+        // store a few records
+        for (int i = 0; i < maxSize; i++) {
+            dbstore.onWireReceive(wireEvelope);
+
+            // prevent key violations - timestamp only has a 1 ms resolution
+            try {
+                Thread.sleep(1);
```
Comment (nicolatimeus) [inline kura/test/org.eclipse.kura.wire.h2db.component.provider.test/src/main/java/org/eclipse/kura/internal/wire/h2db/store/test/H2DbWireRecordStoreTest.java]: This sleep shouldn't be needed anymore

**Review by nicolatimeus — COMMENTED** [review]
I think we can remove the sleeps below, but the PR seems fine, it can also be merged as it is now.

**Comment by nicolatimeus** [comment]
Also please squash the commits if you perform the change. Otherwise I can squash them before merging.

**Comment by markoer** [comment]
Done.

**Pull request merged**